modules/persistenceService/src/main/resources/db/migration/V0__CREATE_USER_TABLE.sql

```diff
@@ -0,0 +1,5 @@
+create table user(
+    id uuid PRIMARY KEY,
+    username VARCHAR(200) UNIQUE NOT NULL
+    name     VARCHAR(200)
```

Comment:
Just a note, for the latest versions of Postgres, specifying the varchar size doesn't provide any performance improvement